### ISSUE TITLE
RCP and SCP

### ISSUE THREAD

**Issue by xinyixinyijiang** [xqwen/fastenloc#14]
Hi,

This could be a stupid question.

I have used fastenloc to perform colocalization of fine-mapped GWAS and fine-mapped GTExv8 data (all tissues). I read this in enloc paper: 
"we propose computing a regional colocalization probability, or RCP, by summing up the SNP-level colocalization probabilities :(SCPs) of correlated SNPs within an LD block that harbors a single GWAS association signal."

But when I sum up the SCP (in SNP-level colocalization result file) of one specific signal in one specific tissue, it is smaller than the RCP I see in the Signal-level colocalization result file. Is my understanding of RCP and SCP wrong? 

Many thanks in advance.

**Comment by xinyixinyijiang** [comment]
In addition, I have some signals having RCP > 1, which seems kinda strange to me. I used the fine-mapped gtexv8 result (from dapg) from this link (https://github.com/hakyimlab/gtex-gwas-analysis), where the data should also be performed from your lab I think. I reformed the gtexv8 fine-mapping res so can be used for fastenloc. Is there any possibility the strange output is because of the reformating?
